ts/src/core/ent.ts

```diff
@@ -1332,37 +1420,30 @@ export function createRowSync(
 
 export function buildUpdateQuery(
   options: EditRowOptions,
-  id: ID,
   suffix?: string,
 ): [string, any[], any[]] {
   let valsString: string[] = [];
   let values: any[] = [];
   let logValues: any[] = [];
-  const dialect = DB.getDialect();
 
   let idx = 1;
   for (const key in options.fields) {
-    values.push(options.fields[key]);
+    const val = options.fields[key];
+    values.push(val);
     if (options.fieldsToLog) {
       logValues.push(options.fieldsToLog[key]);
     }
-    if (dialect === Dialect.Postgres) {
-      valsString.push(`${key} = $${idx}`);
-      idx++;
-    } else {
-      valsString.push(`${key} = ?`);
-    }
+    valsString.push(clause.Eq(key, val).clause(idx));
```

Comment:
TODO this is wrong.

leads to `foo IS NULL` queries when set to null

TODO add test for setting to null correctly